tools/packaging/boms/windows.json

```diff
@@ -3411,6 +3427,22 @@
     "Pattern": "Modules/PSDiagnostics/PSDiagnostics.psm1",
     "FileType": "Product"
   },
+  {
+    "Pattern": "Modules/Microsoft.PowerShell.PSResourceGet/dependencies/*.dll",
+    "FileType": "Product"
+  },
+  {
+    "Pattern": "Modules/Microsoft.PowerShell.PSResourceGet/Microsoft.PowerShell.PSResourceGet.dll",
+    "FileType": "Product"
+  },
+  {
+    "Pattern": "Modules/Microsoft.PowerShell.PSResourceGet/Microsoft.PowerShell.PSResourceGet.psd1",
+    "FileType": "Product"
+  },
+  {
+    "Pattern": "Modules/Microsoft.PowerShell.PSResourceGet/PSGet.Format.ps1xml",
+    "FileType": "Product"
```

Comment:
Product in this context, means that Powershell should be signing them.  From PowerShell's point of view, these are non-product files.

Reply:
@TravisEz13 just updated, do the linux and mac boms look good?

Reply:
linux and mac boms aren't used.  Don't worry about them.